docs/tutorial.md

```diff
@@ -173,7 +173,7 @@ class TicTacToeBoard extends React.Component {
     let winner = '';
     if (this.props.ctx.gameover) {
       winner =
-        this.props.ctx.gameover.winner !== undefined ? (
+        this.props.ctx.gameover.draw  ? (
```

Comment:
Isn't this going to say "Draw!" when `this.props.ctx.gameover.draw` is `false`?

Reply:
Yes you're totally right. I have fixed it.